internal/asm/f32/util_test.go

```diff
@@ -9,11 +9,28 @@ import (
 	"testing"
 )
 
+const (
+	msgRes   = "%v: unexpected result Got: %v Expected: %v"
+	msgVal   = "%v: unexpected value at %v Got: %v Expected: %v"
+	msgGuard = "%v: Guard violated in %s vector %v %v"
+	ε        = 1e-5
+)
+
 var (
 	nan = float32(math.NaN())
 	inf = float32(math.Inf(1))
 )
 
+func within(x, y float32) bool {
```

Comment:
Make it same as the `f64` version, that is, `return same(a, b) || math.Abs(a-b) <= epsilon`?
Also please add a function comment.